### ISSUE TITLE
[CoE Starter Kit - QUESTION] CoE not updating sharing details when app not modified?

### ISSUE THREAD

**Issue by HenningStrand** [microsoft/coe-starter-kit#7975]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

We are doing a cleanup initiative towards app owners to reduce too high sharing numbers for certain apps. We noticed we do not see updates to the CoE reports for these apps. Seems like only changing sharing details does not change the modified date for an app and this is probably why the app is not being picked up by the daily runs of the CoE flows. Is this a correct assumption and that we either have to ask them to modify something else or run a full inventory in order to pick up sharing changes? 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.17

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
That is correct. There are some activities (like sharing) that do not change the last-modified time stamp. As a result the normal inventory flows will not pick it up.

We do have, instead, a set of long running flows which walk the inventory to find things like this. I loosely refer to them as the "Clean up Flows" 
For sharing in particular you can use this one and its helper:
- CLEANUP - Admin | Sync Template v3 (Power Apps User Shared With)
- CLEANUP HELPER - Power Apps User Shared With

Note that it has a few limitations
- Canvas only, we are not able to get this information for Model Driven Apps today. 
- When shared with a group count does not include sub groups people (fixed for April, see https://github.com/microsoft/coe-starter-kit/issues/7738)
